synthtool/sources/git.py

```diff
@@ -141,3 +149,29 @@ def extract_commit_message_metadata(message: str) -> Dict[str, str]:
         metadata[key] = value.strip()
 
     return metadata
+
+
+PRECLONE_MAP_HELP = """
+A preclone map file contains a list of git repos that have already been cloned.
+Its contents are json and look like this:
+{
+  "http://github.com/googleapis/nodejs-vision.git": "/tmp/nodejs-vison",
+  "http://github.com/googleapis/googleapis": "/tmp/googleapis"
```

Comment:
nit: should these examples use `https`?

Reply:
These examples no longer exist because it's documented preconfig.proto.